python-checks/src/main/java/org/sonar/python/checks/PrintStatementUsageCheck.java

```diff
@@ -32,14 +36,35 @@
 @Rule(key = "PrintStatementUsage")
 public class PrintStatementUsageCheck extends PythonCheck {
 
+  public static final String MESSAGE = "Replace print statement by built-in function.";
+
   @Override
   public void initialize(Context context) {
     context.registerSyntaxNodeConsumer(PyElementTypes.PRINT_STATEMENT, ctx -> {
       List<PyElement> expressions = PsiTreeUtil.getChildrenOfAnyType(ctx.syntaxNode(), PyExpression.class, PyPrintTarget.class);
       if (expressions.size() == 1 && expressions.get(0) instanceof PyParenthesizedExpression) {
         return;
       }
-      ctx.addIssue(ctx.syntaxNode().getFirstChild(), "Replace print statement by built-in function.");
+      ctx.addIssue(ctx.syntaxNode().getFirstChild(), MESSAGE);
+    });
+
+    context.registerSyntaxNodeConsumer(PyElementTypes.BINARY_EXPRESSION, ctx -> {
```

Comment:
Could you please add a comment explaining why we need this?

Reply:
Done.